### PR TITLE
RavenDB-7276

### PR DESCRIPTION
**Pull request by arekpalinski**

### CODE REVIEW

File: src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSQLModel.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ import ongoingTask = require("models/database/tasks/ongoingTaskModel");
 
 class ongoingTaskSQLModel extends ongoingTask {
 
-    sqlProvider = ko.observable<string>();
-    sqlTable = ko.observable<string>();
-    sqlTableText: KnockoutComputed<string>;
+    destinationServer = ko.observable<string>();
+    destinationDatabase = ko.observable<string>();
```
Comment (Danielle9897) [inline src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSQLModel.ts:8]: Need to update matching fields in *ongoingTasks.html* as well

Comment (arekpalinski) [reply src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSQLModel.ts:8]: Fixed in 6699dcfeaba7. Can you have a look?

Comment (arekpalinski) [reply src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSQLModel.ts:8]: Those changes are not enough. Working on that.

Comment (arekpalinski) [reply src/Raven.Studio/typescript/models/database/tasks/ongoingTaskSQLModel.ts:8]: Actually, it should be fine now

**Comment by arekpalinski** [comment]
@ayende merged to fix studio compilation

**Pull request merged**